std_msgs/Header header
float64 vel1
float64 vel2
float64 vel3
float64 vel4
geometry_msgs/Quaternion orientation
geometry_msgs/Vector3 angular_velocity